stl/inc/ranges

```diff
@@ -3267,8 +3267,8 @@ namespace ranges {
     template <input_range _Vw, forward_range _Pat>
         requires (view<_Vw> && view<_Pat>
             && indirectly_comparable<iterator_t<_Vw>, iterator_t<_Pat>, _RANGES equal_to>
-            && (forward_range<_Vw> || _Tiny_range<_Pat>)
-    class lazy_split_view;
+            && (forward_range<_Vw> || _Tiny_range<_Pat>))
+    class split_view;

```

Comment:
This seems like a mistake made while resolving merge conflicts. Ditto below

Reply:
Thanks a lot!